auto_dev/utils.py

```diff
@@ -284,6 +284,13 @@ def remove_suffix(text: str, suffix: str) -> str:
     return text[: -len(suffix)] if suffix and text.endswith(suffix) else text
 
 
+def load_basic_aea_config():
```

Comment:
We definitley dont need a whole function for this :p

Reply:
TBH i like this as opposed to grabbing the first index everywhere, happy to remove though

Reply:
you can do this;

```
base_config, _ = load_aea_config()
```
Add a default vaule of with_override=True instead of a whole function?

Then have mutating outputs though which isnt nice